treeherder/etl/management/commands/ingest.py

```diff
@@ -21,6 +23,8 @@
                                                       handleMessage)
 from treeherder.model.models import Repository
 
+GITHUB_API = "https://api.github.com"
```

Comment:
Shouldn't this be in a settings or config file?

Reply:
Maybe... If I find it needed anywhere else.

Reply:
I've looked around in the code and I don't think we gain much by centralizing the value. I've made it part of the URL in the code being used below.